src/scss/components/navigation/_table-of-contents.scss

```diff
@@ -0,0 +1,85 @@
+.table-of-contents {
+  @include components-container-max-width();
+  margin-left: 0;
+  margin-right: 0;
+
+  @include breakpoint($breakpoint-m) {
+    @include component-side-padding();
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .has-sidebar & {
+    @include component-negative-side-margin;
+  }
+
+  // In rare cases TPR Service page can be rendered without sidebar.
+  // In these cases remove the left and right paddings from table of contents.
+  .service-header__container:not(.has-sidebar) & {
+    @include breakpoint($breakpoint-m) {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+}
+
+.table-of-contents__container {
+  @include component-margin-top();
+  background-color: $hdbt-color-white;
+  border-left: 8px solid var(--hdbt-color-gold--primary);
+  max-width: $content-width-max;
```

Comment:
If the max-width for paragraphs inside this element is `$content-width-max` (=860px), adding this max-width here when we have padding also added means that the paragraphs are not the wanted width, but 804px instead.